### PR TITLE
Appdrusti 6 new

### PR DESCRIPTION
**Pull request by BOSSHK-SPEC**

### CODE REVIEW

File: lib/LoginAndRegistration.dart [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import 'dart:convert';
-
+import 'package:flutter_session/flutter_session.dart';
```
Comment (sahana-drusti) [inline lib/LoginAndRegistration.dart:2]: remove flutter-session

File: lib/LoginAndRegistration.dart [hunk 2]
```diff
@@ -253,9 +264,15 @@ class LoginAndRegisterationPageState extends State {
                                 validator: (val) {
                                   if (val == null || val.isEmpty) {
                                     return 'Please enter registration number';
+                                  } else if (!validRegNo) {
```
Comment (sahana-drusti) [inline lib/LoginAndRegistration.dart:267]: error message should should be in red

File: pubspec.lock [hunk 3]
```diff
@@ -50,23 +50,77 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.0.3"
```
Comment (sahana-drusti) [inline pubspec.lock:52]: remove this file

File: pubspec.yaml [hunk 4]
```diff
@@ -30,6 +30,7 @@ dependencies:
   cupertino_icons: ^1.0.2
```
Comment (sahana-drusti) [inline pubspec.yaml:30]: remove flutter-session dependency as it is not used anywhere yet

File: lib/LoginAndRegistration.dart (hunk 1 above)
Comment (BOSSHK-SPEC) [reply lib/LoginAndRegistration.dart:2]: done

File: lib/LoginAndRegistration.dart (hunk 2 above)
Comment (BOSSHK-SPEC) [reply lib/LoginAndRegistration.dart:267]: done

File: pubspec.yaml (hunk 4 above)
Comment (BOSSHK-SPEC) [reply pubspec.yaml:30]: ok

File: pubspec.lock (hunk 3 above)
Comment (BOSSHK-SPEC) [reply pubspec.lock:52]: ok

**Pull request merged**